.github/workflows/ci-cd.yml

```
@@ -85,11 +85,35 @@ jobs:
       - name: Run backend tests
         run: pnpm --filter @flaira/backend run test    
 
-  build-and-push:
+  push-db-migrations:
     needs: [lint, typecheck, backend-tests]
     if: github.ref == 'refs/heads/main'
     runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)